docs/SRS/SRS.tex

```diff
@@ -487,25 +488,50 @@ \subsubsection{Work Orders}
 \section{Non-functional Requirements}
 
 \subsection{Look and Feel Requirements}
+\begin{enumerate}
+	\item [{LF}1.] The system shall adjust and scale to fit the physical screen size
+	\item [{LF}2.] The system shall have font and colours that will allow users to easily read the text
+\end{enumerate}
 
 \subsection{Usability and Humanity Requirements}
+\begin{enumerate}
+	\item [{UH}1.] The system shall be easy for any person to use all features
```

Comment:
I don't think this is verifiable. The example in the rubric, "The system shall be user friendly is not verifiable".